projects/plugins/jetpack/extensions/blocks/sharing-button/class-sharing-source-block.php

```diff
@@ -114,8 +114,7 @@ public function get_share_title( $post_id ) {
 		 * @param int $post_id Post ID.
 		 * @param int $this->id Sharing ID.
 		 */
-		$title = apply_filters( 'sharing_title', $post->post_title, $post_id, $this->id );
-
+		$title = $post ? apply_filters( 'sharing_title', $post->post_title, $post_id, $this->id ) : '';
```

Comment:
Would it be worth having a more explicit check?

```suggestion
		$title = $post instanceof WP_Post
			? apply_filters( 'sharing_title', $post->post_title, $post_id, $this->id )
			: '';
```

Although that will most likely mean you'll have to move the docblock closer to the filter in that statement.

Reply:
Good idea, added in cb63c67 (put it all on one line as it isn't too long, presumably that won't cause issues for the docblock?)